client/mysqlbinlog.cc

```diff
@@ -1942,6 +1952,11 @@ get_one_option(const struct my_option *opt, const char *argument, const char *fi
   case 'T':
     one_table= 1;
     break;
+  case 'L':
```

Comment:
Is it worth supporting a argument=`-N` output to file descriptor syntax and use `fdopen(N, ...)` as as a open method. This would enable the use of coprocesses and other scripted contructs that don't require a filesystem fifo to perform live interaction.

Reply:
Right, that's a possible case. But not sure if we need to support fdopen now. Maybe we can extend to support it later if there's requirement for that?

Reply:
Such argument feels to be a good idea for the most of the mariadb client programs, and then
should not be implemented solely to this one.